libvmaf/include/libvmaf/libvmaf.h

```diff
@@ -253,6 +253,25 @@ int vmaf_score_at_index_model_collection(VmafContext *vmaf,
 int vmaf_feature_score_at_index(VmafContext *vmaf, const char *feature_name,
                                 double *score, unsigned index);
 
+typedef struct VmafMetadataConfig {
+    void (*callback)(void *, const char *, double);
```

Comment:
Rather than passing two extra parameters the callback `(char*, double)` let's pass another struct. For now, the struct can contain these two `(char*, double)` but it is much more flexible if we need to add anything in the future, since we can do so without breaking API.